prbench/src/prbench/envs/dynamic3d/objects.py

```diff
@@ -402,14 +406,28 @@ def get_position(self) -> NDArray[np.float32]:
         """
         return np.array(self.position)
 
-    def get_orientation(self) -> NDArray[np.float32]:
+    def get_orientation(self) -> list[float]:
         """Get the fixture's orientation.
 
         Returns:
-            Orientation as quaternion [w, x, y, z] array
+            Orientation as quaternion [w, x, y, z] list
+        """
+        return utils.convert_yaw_to_quaternion(self.yaw)
+
+    @staticmethod
+    @abc.abstractmethod
+    def get_bounding_box_from_config(
```

Comment:
I think we should make this consistent with `get_bounding_box_dimensions`. the main difference between the two right now is that `get_bounding_box_dimensions` (in MujocoObject) only gives you dimensions, whereas this is meant to give the bb in the world frame. it looks like almost all the uses of this method so far use `pos = np.array([0, 0, 0])`, which indicates that maybe dimensions are all we need? we could always transform the dimensions into the world frame externally if necessary. if you would prefer to keep this in the world frame, I think there is a case to be made for inputting not just a pos but a full pose. we have had some bugs in the past related to rotations and aabbs

Reply:
Good point. I prefer to use bounding box coordinates in the world frame. The reason being that, this way we do not need to assume that the object is symmetric around its local origin. That is, if the object's local frame is aligned with the world frame, it's bounding box can still be anywhere in the world, and dimensions are hence not sufficient.

I have created an issue [here](https://github.com/Princeton-Robot-Planning-and-Learning/prpl-mono/issues/165) and will fix this in a PR soon.